package/main/daml/Daml.Finance.Claims/daml.yaml

```diff
@@ -13,8 +13,8 @@ data-dependencies:
   - .lib/daml-finance/ContingentClaims.Lifecycle/2.0.0/contingent-claims-lifecycle-2.0.0.dar
   - .lib/daml-finance/Daml.Finance.Interface.Claims/3.0.0/daml-finance-interface-claims-3.0.0.dar
   - .lib/daml-finance/Daml.Finance.Interface.Instrument.Base/3.0.0/daml-finance-interface-instrument-base-3.0.0.dar
-  - .lib/daml-finance/Daml.Finance.Interface.Instrument.Types/0.99.0.20230927.1/daml-finance-interface-instrument-types-0.99.0.20230927.1.dar
-  - .lib/daml-finance/Daml.Finance.Interface.Lifecycle/2.99.0.20230927.1/daml-finance-interface-lifecycle-2.99.0.20230927.1.dar
+  - .lib/daml-finance/Daml.Finance.Interface.Instrument.Types/1.0.0/daml-finance-interface-instrument-types-1.0.0.dar
```

Comment:
I was checking the other PR, would it make sense to stay on a snapshot version?

Reply:
Discussed it with @markus-da , and we thought it would be fine to move to the target version now considering that it most likely won't be any snapshot release until then. Worst case we could revert to the snapshot version.

Reply:
Mm, this makes it cumbersome to test the quickstarter shipped with the `daml` repo before the release (we would need to have our packages released first). It might be worth checking with @georg-da what his preference his

Reply:
there is now a `daml-finance/docs/code-samples/scripts/get-local-dependencies.sh` to test this locally, not sure it is worth doing a snapshot release for the sole purpose of checking the quickstarter?

Reply:
Would that be a pre-release for the sole purpose of testing the quickstarter? I believe that is not part of the release process, but we can always add it if it is worth doing.
With the script that I added some time ago it is possible to test parts of the flow with locally compiled files, but not end-to-end.

Reply:
Yup, that's something I used while performing the `2.7` release for extra safety

Reply:
Ok, I will roll the versions back to the snapshot versions (but with new dates). Then we can decide later if we want to do another snapshot release before the 2.8 release.

Reply:
I think that makes sense, thanks Johan! Maybe it's not too late, but you can save the target versions in a stash / temporary commit so that they can be re-used when we want to make the release